src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -84,6 +85,7 @@ ResponseEntity<Resource<?>> suggest(@RequestBody final ObjectNode suggestion,
         return ResponseEntity.ok().body(assembler.toFullResource(suggestedProject));
     }
 
+    @PreAuthorize("#project.submissionEnvelope == null || #project.submissionEnvelope.inEditableState")
```

Comment:
👍

Reply:
actually, we may want to check all the submissions linked to the project

Reply:
Ah true, damn